chapter2.Rmd

```diff
@@ -108,32 +112,33 @@ Before doing a first analysis, you decide to first collect all the winnings and
 
 For `poker_vector`: 
 
-- On Monday you won $140
-- Tuesday you lost $50
-- Wednesday you won $20 
-- Thursday you lost $120
-- Friday you won $240
+- On Monday you won $140.25
+- Tuesday you lost $50.50
+- Wednesday you won $20.25
+- Thursday you lost $120.00
+- Friday you won $240.00
 
 For `roulette_vector`: 
 
-- On Monday you lost $24
-- Tuesday you lost $50
+- On Monday you lost $24.75
+- Tuesday you lost $50.25
 - Wednesday you won $100
-- Thursday you lost $350
+- Thursday you lost $350.50
 - Friday you won $10
 
 You only played poker and roulette, since there was a delegation of mediums that occupied the craps tables. To be able to use this data in R, you decide to create the variables `poker_vector` and `roulette_vector`.
```

Comment:
Medium = psychic, and I think the correct collective noun is a coven. If we're doing bad jokes, I quite like "You only play poker and roulette, ever since the misunderstanding at the craps table."

Anyway, I just removed the sentence for now in e00b5c63f3d3df43f0e8ad73311eb40a7ee9ea90